dev-python/ciso8601/ciso8601-2.1.3.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+PYTHON_COMPAT=( python3_{7,8} )
+
+inherit distutils-r1
+
+DESCRIPTION="Fast ISO8601 date time parser for Python written in C"
+HOMEPAGE="https://github.com/closeio/ciso8601 https://pypi.org/project/ciso8601/"
+SRC_URI="mirror://pypi/${P:0:1}/${PN}/${P}.tar.gz"
```

Comment:
We don't really like mirror:// links. If possible use a full link like SRC_URI="https://github.com/closeio/ciso8601/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"

Reply:
thank you for the detailed explanation

Reply:
You are welcome :)